src/components/Frontpage/index.js

```diff
@@ -217,52 +243,69 @@ const Frontpage = ({ location, lang }) =>
             )}
           </div>
         </div>
+
+        {/* Calendar section*/}
+        <div className={cx('content')} >
+          {/* Title */}
+          <h2 id="sections_intro">
+            <Translate>
+              <Swedish>Evenemangskalender</Swedish>
+              <English>Event Calendar</English>
+            </Translate>
+          </h2>
+
+          {/* Calendar */}
+          <Calypso type='event'>
```

Comment:
After some investigation, to make the calendar load all events it seems like we should use
```
          <Calypso search="itemType='EVENT'">
```
here. (This makes the component use `https://calypso.datasektionen.se/api/list?itemType=EVENT` instead of `https://calypso.datasektionen.se/api/event`)
However, the result from that endpoint is paginated (which is good so we don't load all our event history at each pageload), so it would not be a drop in replacement. So it can probably wait for another PR.

Reply:
I think the best solution to this would be to add an endpoint to calypso that allows to query for events in a time range, that should be used here. But that could be changed later since the changes to bawang would be very small.

Reply:
I agree, and also think that this can wait for another PR.